docs/styles.css

```
@@ -8,3 +8,20 @@ body {
   word-break: keep-all;
   word-wrap: break-word;
 }
+
+.navbar-github-link::before {
+  content: '';
+  width: 24px;
+  height: 24px;
+  display: flex;
+  transition: opacity 0.2s ease-in-out;
+  background: url(../static/img/github.svg) no-repeat center;
+}
+
+.navbar-github-link:hover::before {
+  opacity: 0.6;
+}
+
+[data-theme='dark'] .navbar-github-link::before {
+  background: url(../static/icons/github-dark.svg) no-repeat center;
```

```suggestion
  background: url(../img/github.svg) no-repeat center;
```

I think it will work by [static-assets](https://docusaurus.io/docs/next/static-assets)

In docusaurus.config.js baseUrl is just "/" 
so the image /static/img/github-dart.svg will be served ad /img/github-dart.svg.

I think this will fix build error but if fail we have to find other solution!